src/OpenLoco/src/Windows/Construction/ConstructionTab.cpp

```diff
@@ -1829,7 +1828,7 @@ namespace OpenLoco::Ui::Windows::Construction::Construction
 
         if (_constructionHover == 1)
         {
-            if (!Input::isToolActive(WindowType::construction, self.number) || _toolWidgetIndex != widx::construct)
+            if (!Input::isToolActive(WindowType::construction, self.number) || ToolManager::getToolWidgetIndex() != widx::construct)
```

Comment:
This is potentially the source of https://github.com/OpenLoco/OpenLoco/issues/1281. I want to do a few little tests this was incorrectly accessing the toolwidgetindex before.

Reply:
Yeah i'm pretty sure this is the cause of #1281.